alpaca/data/requests.py

```diff
@@ -546,3 +550,28 @@ class NewsRequest(NonEmptyRequest):
     include_content: Optional[bool] = None
     exclude_contentless: Optional[bool] = None
     page_token: Optional[str] = None
+
+
+# ############################## CorporateActions #################################### #
+
+
+class CorporateActionsRequest(NonEmptyRequest):
+    """
+    This request class is used to submit a request for corporate actions data.
+    ref. https://docs.alpaca.markets/reference/corporateactions-1
+
+    Attributes:
+        symbols (Optional[List[str]]): The list of ticker identifiers.
+        types (Optional[List[CorporateActionsType]]): The types of corporate actions to filter by. (default: all types)
+        start (Optional[date]): The inclusive start of the interval. Format: YYYY-MM-DD. (default: current day)
+        end (Optional[date])): The inclusive end of the interval. Format: YYYY-MM-DD. (default: current day)
+        limit (Optional[int]): Upper limit of number of data points to return. (default: 1000)
+        sort (Optional[Sort]): The chronological order of response based on the timestamp. Defaults to ASC.
+    """
+
+    symbols: Optional[List[str]] = None
+    types: Optional[List[CorporateActionsType]] = None
+    start: Optional[date] = None
+    end: Optional[date] = None
+    limit: Optional[int] = 1000
```

Comment:
This should default to None, as it sets the total limit of the request (not the page size)!

Reply:
I prefer to set default limit as user should explicitly request larger number of response.
But yeah, we hide page_token in current design. Agree. reasonable to set None.